### ISSUE TITLE
Null safety

### ISSUE THREAD

**Issue by chrisgarber** [you-dont-need/You-Dont-Need-Lodash-Underscore#164]
First of all thanks so much for this library, love using this plugin to remove old lodash uses.

I wanted to quickly broach the topic of null safety; many functions in lodash, e.g. _.keys, _.entries are null safe. The same safety can be achieved by using the native methods like so: `Object.keys(value || {})` As this could be breaking for those of us converting, it would be great if this were mentioned in the error messages and the readme. I'm happy to do a PR with a draft of these changes, but as this is an issue that effects a number of functions I figured you might want some input over how to change the text.

I propose something like `"Object.keys(value || {})"` in rules.json, and maybe `Consider using the native Object.keys as Object.keys(value || {})` in the error messages?

**Comment by stevemao** [comment]
I think putting it in `rules.json` is too wordy. Maybe we should add a note in readme?

**Comment by chrisgarber** [comment]
Sure, do we want examples for each instance where the null safety applies? One large general note?

**Comment by stevemao** [comment]
I think a large general note is enough. What do you think?

**Comment by stevemao** [comment]
Thanks @chrisgarber , PR welcome :)

**Comment by cjtaylor1990** [comment]
@stevemao If this issue is still up for grabs, I can do it at the same time I handle #110

**Comment by stevemao** [comment]
@cjtaylor1990 Thanks buddy :)

**Comment by cjtaylor1990** [comment]
Issue handled in #223
